auslib/db.py

```diff
@@ -1764,6 +1765,56 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None):
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction)
 
 
+class Shutoffs(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('shutoffs', metadata,
+                           Column('shutoff_id', Integer, primary_key=True,
+                                  autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75)),
+                           Column('mapping', String(100)),
+                           UniqueConstraint('product', 'channel')
+                           )
+        AUSTable.__init__(self, db, dialect, history=False, versioned=False)
+
+    def getShutoff(self, transaction=None, **columns):
```

Comment:
I don't think there's any reason to catch extra kwargs here. Is there a reason you're doing this instead of having "product" and "channel" as keyword arguments?

If you were overriding select() you'd have to do this, but since you're writing a custom method there's no reason you need to match select()'s interface.